### DISCUSSION TITLE
Always zero NOPM

### DISCUSSION THREAD

**Discussion by juanludlf** [TPC-Council/HammerDB#300] [Q&A] [answered]
I always get 0 NOPM in test results. 
The target database is AWS Aurora Serverless (MySQL compatible edition).
I'm limiting the `MAXITEMS` field to 1000 (instead of 1000000) in `mysqloltp.tcl` file.
Total transactions per virtual user is 100000.

![image](https://user-images.githubusercontent.com/6860752/146366951-da7e8da6-4612-4968-abc9-1584276fd066.png)

What am I doing wrong? Will increasing the "MAXITEMS" field solve this problem?
Thanks.

**Comment by sm-shaw** [comment]
Converting to discussion topic. Not an issue as source has been modified.

**Comment by sm-shaw** [comment, marked as answer]
Yes, certainly modifying the build could have an impact on whether the build proceeds up to creating the stored procedures and whether they run correctly in the driver. The first thing to do is to set "Exit on MySQL Error" in the driver script options and see if it reports an error, by default errors are suppressed. If it does, then set MAXITEMS back to the default and re-run the schema build. 

If you want to vary the amount of data created, use the number of warehouses for this as the schema is scaled based on the selected warehouse_id.

**Reply by juanludlf** [reply]
Thank you for your response @sm-shaw 
I have already checked the "Exit on MySQL error" checkbox, no errors were thrown. I will try to reset MAXITEMS to its original value.

**Reply by sm-shaw** [reply]
Also try running the test script with a small number of virtual users. This prints the output to the screen for this type of troubleshooting.
